### PR TITLE
Add Maintainers Guidelines

### PR DESCRIPTION
**Pull request by tkuhrt**
Signed-off-by: Kuhrt, Tracy A <tracy.a.kuhrt@accenture.com>

### CODE REVIEW

**Review by dcmiddle — APPROVED** [review]

File: docs/MAINTAINERS-guidelines.md
Comment (swetharepakula) [inline docs/MAINTAINERS-guidelines.md:10]: Would adding the rocket chat id be useful for the active maintainers?

**Review by swetharepakula — APPROVED** [review]

File: docs/MAINTAINERS-guidelines.md
Comment (lehors) [reply docs/MAINTAINERS-guidelines.md:10]: I agree with Swetha, I find it difficult to contact people when all I have is their github id. I would favor adding both the RC id and an email address, with a requirement to have at least one or the other provided.

File: docs/MAINTAINERS-guidelines.md
Comment (lehors) [inline docs/MAINTAINERS-guidelines.md:2]: Some repositories currently have a MAINTAINERS.rst file instead. I think we should allow that variation.

File: docs/MAINTAINERS-guidelines.md
Comment (lehors) [inline docs/MAINTAINERS-guidelines.md:40]: an emeritus

File: docs/MAINTAINERS-guidelines.md
Comment (lehors) [inline docs/MAINTAINERS-guidelines.md:43]: These are not all MAINTAINERS files though. You might want to just say "Examples" to cover all cases.

File: docs/MAINTAINERS-guidelines.md
Comment (lehors) [inline docs/MAINTAINERS-guidelines.md:25]: I'm wondering whether we should allow for that info to be elsewhere and merely have a link to it in that case because several projects have that info in their docs today. On the other hand, the change this involves is simple and a link could be made the other way around...

File: docs/MAINTAINERS-guidelines.md
Comment (tkuhrt) [reply docs/MAINTAINERS-guidelines.md:10]: Isn’t your LFID the same as your Rocket.Chat ID?

File: docs/MAINTAINERS-guidelines.md
Comment (tkuhrt) [reply docs/MAINTAINERS-guidelines.md:2]: Agree

File: docs/MAINTAINERS-guidelines.md
Comment (tkuhrt) [reply docs/MAINTAINERS-guidelines.md:43]: Isn’t this what title says? Not sure I understand comment.

File: docs/MAINTAINERS-guidelines.md
Comment (tkuhrt) [reply docs/MAINTAINERS-guidelines.md:25]: I was hoping the SHOULD gave that out.

File: docs/MAINTAINERS-guidelines.md
Comment (lehors) [reply docs/MAINTAINERS-guidelines.md:10]: Ah, good point. I guess it is. Maybe we could highlight that in having both in the header for dummies like me who don't think about it. :-)

File: docs/MAINTAINERS-guidelines.md
Comment (lehors) [reply docs/MAINTAINERS-guidelines.md:43]: The current title reads "Example MAINTAINERS Files". I'm saying making it merely "Examples" given that the examples provided are for the most part not MAINTAINERS files.

File: docs/MAINTAINERS-guidelines.md
Comment (lehors) [reply docs/MAINTAINERS-guidelines.md:25]: Yeah but the thing is that we do want to be able to get to that info from the MAINTAINERS file. This is often not the case today. So I think adding something stating that this needs to be in the file or linked from it would give better results.

File: docs/MAINTAINERS-guidelines.md
Comment (tkuhrt) [reply docs/MAINTAINERS-guidelines.md:10]: Changes made to reflect. See if this meets the needs.

File: docs/MAINTAINERS-guidelines.md
Comment (tkuhrt) [reply docs/MAINTAINERS-guidelines.md]: Changes made to reflect.

File: docs/MAINTAINERS-guidelines.md
Comment (tkuhrt) [reply docs/MAINTAINERS-guidelines.md]: Corrected

File: docs/MAINTAINERS-guidelines.md
Comment (tkuhrt) [reply docs/MAINTAINERS-guidelines.md]: Done. Thanks for explaining.

File: docs/MAINTAINERS-guidelines.md
Comment (tkuhrt) [reply docs/MAINTAINERS-guidelines.md:25]: We should discuss with rest of @hyperledger/tsc. I was trying to not be too prescriptive with the guidelines.

File: docs/MAINTAINERS-guidelines.md
Comment (swetharepakula) [reply docs/MAINTAINERS-guidelines.md:10]: My rocket chat id differs from my LFID, though I use my LFID to sign into rocket chat.

**Review by christo4ferris — APPROVED** [review]
LGTM now that it includes email etc. Thanks, Tracy!

File: docs/MAINTAINERS-guidelines.md
Comment (tkuhrt) [reply docs/MAINTAINERS-guidelines.md:16]: Thanks, @swetharepakula. I updated.

**Comment by hartm** [comment]
Looks good to me.  Thanks Tracy!

File: docs/MAINTAINERS-guidelines.md
Comment (adecaro) [inline docs/MAINTAINERS-guidelines.md:8]: Is it clear that we recommend at least those fields and that it is not forbidden to add other fields?

File: docs/MAINTAINERS-guidelines.md
Comment (adecaro) [inline docs/MAINTAINERS-guidelines.md:14]: I would like to make sure that the maintainers fell free to choose the format they prefer especially if they plan to use GitHub hooks as David was suggesting during the TSC call.

**Comment by ryjones** [comment]
I think you have the approvals needed for merge

File: docs/MAINTAINERS-guidelines.md
Comment (tkuhrt) [reply docs/MAINTAINERS-guidelines.md:15]: Agree. That is why it is a "suggested format"

File: docs/MAINTAINERS-guidelines.md
Comment (tkuhrt) [reply docs/MAINTAINERS-guidelines.md:9]: This is only a recommended list of fields. Do you have a suggestion on re-wording?

File: docs/MAINTAINERS-guidelines.md
Comment (christo4ferris) [reply docs/MAINTAINERS-guidelines.md:32]: I'm sympathetic to Tracy's POV here

**Review by christo4ferris — APPROVED** [review]
LGTM

**Review by troyronda — APPROVED** [review]

**Review by mastersingh24 — APPROVED** [review]

**Pull request merged**
